tests/testthat/test-plot-grid.R

```diff
@@ -0,0 +1,52 @@
+test_that("plots grid is rendered correctly", {
+  skip_if_not_installed("vdiffr")
+  skip_if_not_installed("ggplot2")
+  skip_if_not_installed("tlf")
+  skip_if(getRversion() < "4.1")
```

Comment:
does it work only with R>=4.1 or why `skip_if(getRversion() < "4.1")` ?

Reply:
Yes, R graphics engine changed in `4.1`, so the plot SVGs won't be the same for R versions prior to that.

Reply:
ok